modules/installation-openstack-sr-iov-requirements.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-nfv-preparing
+
+:_content-type: PROCEDURE
+[id="installation-openstack-sr-iov-requirements_{context}"]
+= Requirements for clusters on {rh-openstack} that use SR-IOV
+
+To use single-root I/O virtualization (SR-IOV) with your deployment, you must meet the following requirements:
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html-single/network_functions_virtualization_planning_and_configuration_guide/index#assembly_sriov_parameters[Plan your RHOSP SR-IOV deployment].
+
+* NICs that you use must be link:https://docs.openshift.com/container-platform/4.10/networking/hardware_networks/about-sriov.html[supported by OCP].
+
+* For each OCP node that will have an attached SR-IOV NIC, your RHOSP cluster must have:
```

Comment:
RHOSP should be `{rh-openstack}` throughout.

Reply:
```suggestion
* For each OCP node that will have an attached SR-IOV NIC, your {rh-openstack} cluster must have:
```